data/dapis/avalanche.json

```diff
@@ -1,5 +1,4 @@
 {
-  "AVAX/USD": "0x098cf42f8ea393f32ec9b788d48eb58d044a51a8793f1d98ceb27d25fccbeb2b",
```

Comment:
Should we also update the policy here: https://github.com/api3dao/operations/blob/update-dapis/data/policies/avalanche/dataFeeds/0x741c2f6e00b2c07badf4920c8f1f2ebe7bddb2ef-0x098cf42f8ea393f32ec9b788d48eb58d044a51a8793f1d98ceb27d25fccbeb2b.json#L5